install/install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+if [ "$#" -ne 1 ]; then
+    echo "Invalid number of parameters"
+		exit 1
+fi
+
+JSONPATH1="{.spec.replicas}"
+JSONPATH2="{.status.readyReplicas}"
+ZkOprName=zoo-op
+ZkName=zoo
+BkOprName=pr
+BkName=br
+PrOprName=foo
+PrName=bar
+
+if [ "$1" == "1" ]; then
+	helm install ../charts/zookeeper-operator --name $ZkOprName
+	kubectl rollout status deploy/$ZkOprName-zookeeper-operator
+
+	helm install ../charts/zookeeper --name $ZkName
+
+	replicas=$(kubectl get zookeepercluster $ZkName-zookeeper -o jsonpath=$JSONPATH1)
+	ready=$(kubectl get zookeepercluster $ZkName-zookeeper -o jsonpath=$JSONPATH2)
+
+	until [ "$replicas" == "$ready" ]
+	do
+		sleep 1;
+		replicas=$(kubectl get zookeepercluster $ZkName-zookeeper -o jsonpath=$JSONPATH1)
+		ready=$(kubectl get zookeepercluster $ZkName-zookeeper -o jsonpath=$JSONPATH2)
+	done
+
+	ZkSvc=$(kubectl get svc | grep $ZkName- | grep client | head -n1 | cut -d " " -f1)
+
+	helm install ../charts/bookkeeper-operator --name $BkOprName
+	kubectl rollout status deploy/$BkOprName-bookkeeper-operator
+
+	helm install ../charts/bookkeeper --name $BkName --set zookeeperUri=$ZkSvc:2181
```

Comment:
No hardcoded client port. Please pick values from charts...

Reply:
the client port values are fixed and hardcoded within the charts as well within the name of the service. Hence I feel these values can be hardcoded here